### PR TITLE
add more custom icons for model side nav

### PR DESCRIPTION
**Pull request by tbehailu**
Add missing custom icons to ui-library for new model sidenav. 
## Dark mode
Screenshot 2023-11-27 at 11 00 35 AM

## Light mode 

Screenshot 2023-11-29 at 4 35 37 PM

### CODE REVIEW

File: src/icon/Icons.tsx [hunk 1]
```diff
@@ -812,6 +824,38 @@ export const CloudUpload = () => (
 );
 
 //D
+export const Datasets = () => (
```
Comment (mikeldking) [inline src/icon/Icons.tsx:827]:
```suggestion
export const DatabaseOutline = () => (
```
just to name it a bit more semantically

File: src/icon/Icons.tsx [hunk 2]
```diff
@@ -874,6 +918,23 @@ export const DownloadOutline = () => (
   </svg>
 );
 
+export const Drift = () => (
```
Comment (mikeldking) [inline src/icon/Icons.tsx:921]:
```suggestion
export const DriftOutline = () => (
```

File: src/icon/Icons.tsx [hunk 3]
```diff
@@ -932,6 +993,24 @@ export const Edit2Outline = () => (
   </svg>
 );
 
+export const Embeddings = () => (
```
Comment (mikeldking) [inline src/icon/Icons.tsx:996]:
```suggestion
export const EmbeddingsOutline = () => (
```

File: src/icon/Icons.tsx [hunk 4]
```diff
@@ -961,6 +1040,107 @@ export const ExpandOutline = () => (
...
+          fill="white"
+          fillOpacity="0.9"
+      />
+      <rect
+          x="14.5417"
+          y="3.37933"
+          width="2.5"
+          height="3.64864"
+          rx="1.25"
+          fill="white"
```
Comment (mikeldking) [inline src/icon/Icons.tsx:1138]: We need to delete all these "white" hard codings. If you run the the storybook the colors will probably not rotate.

File: src/icon/Icons.tsx [hunk 5]
```diff
@@ -996,6 +1176,30 @@ export const EyeOutline = () => (
...
+          stroke="white"
+          strokeOpacity="0.9"
+          strokeWidth="1.5"
+      />
+      <path
+          fillRule="evenodd"
+          clipRule="evenodd"
+          d="M4.28993 13.218C3.99484 12.9273 3.51641 12.9273 3.22132 13.218C2.92623 13.5087 2.92623 13.98 3.22132 14.2707L5.99212 17.0001L3.22132 19.7295C2.92623 20.0202 2.92623 20.4915 3.22132 20.7822C3.5164 21.0729 3.99484 21.0729 4.28992 20.7822L7.06073 18.0527L9.71152 20.664C10.0066 20.9546 10.485 20.9546 10.7801 20.664C11.0752 20.3733 11.0752 19.902 10.7801 19.6113L8.12934 17.0001L10.7801 14.3889C11.0752 14.0982 11.0752 13.6269 10.7801 13.3362C10.485 13.0456 10.0066 13.0456 9.71151 13.3362L7.06073 15.9474L4.28993 13.218Z"
+          fill="white"
+          fillOpacity="0.9"
```
Comment (mikeldking) [inline src/icon/Icons.tsx:1198]: same here, nix the opacity and color

File: src/icon/Icons.tsx (hunk 1 above)
Comment (tbehailu) [reply src/icon/Icons.tsx]: done

File: src/icon/Icons.tsx (hunk 5 above)
Comment (tbehailu) [reply src/icon/Icons.tsx]: done

File: src/icon/Icons.tsx (hunk 3 above)
Comment (tbehailu) [reply src/icon/Icons.tsx]: done

File: src/icon/Icons.tsx (hunk 2 above)
Comment (tbehailu) [reply src/icon/Icons.tsx]: done

File: src/icon/Icons.tsx (hunk 4 above)
Comment (tbehailu) [reply src/icon/Icons.tsx]: took a minute but removed all the fills! had to regenerate some of these icons since figma automatically includes it on export

**Review by mikeldking — APPROVED** [review]

**Pull request merged**